tests/ember-intl/tests/integration/helpers/format-date-range/input-is-falsy-test.ts

```diff
@@ -25,7 +25,6 @@ module(
     test('input is an empty string', async function (assert) {
       await render(hbs`
         <div data-test-output>
-          {{! @glint-expect-error }}
```

Comment:
This was giving me lint errors, not sure why.
I removed it and everything seems to work fine. Tests and lint pass.

Reply:
Until `@formatjs/intl@3.1.0`, its `formatDateTimeRange()` didn't allow strings to be passed, so we had to tell Glint to ignore the type errors. Now that strings can be passed, Glint is letting us know that we don't need this override anymore.